netbox/project-static/styles/overrides/_tabler.scss

```diff
@@ -35,12 +35,85 @@ pre {
   display: inline-block;;
 }
 
-table a {
-  // Adjust table anchor link contrast as not enough contrast in dark mode
-  filter: brightness(110%);
+table {
+  a {
+    color: $dark-teal;
```

Comment:
This can probably be removed as it appears to be redundant.

Reply:
removed.